### PR TITLE
Added Milestone #1 - Dhanvi Ganti

### PR DESCRIPTION
**Pull request by drdonv**
forgot to use this repo instead of the other website one when coding. Added blog.html, blog.ts, blog.js, gitignore, and all others.

## Developer: {Dhanvi Ganti}

Closes #3 

### Pull Request Summary

Added the blog typescript stuff.
### Pull Request Checklist

- [ ] Code is neat, readable, and works
- [ ] Comments are appropriate
- [ ] The commit messages follows our [guidelines](https://h4i.notion.site/Conventional-Commits-593452ad1179489399ad3bd696ef772a)
- [ ] The developer name is specified
- [ ] The summary is completed
- [ ] Ask for a review in communication channels

### Screenshot
Screenshot 2023-10-23 at 9 47 57 PM
s/Screencast

### CODE REVIEW

**Comment by jlaksana** [comment]
It does not seem like you properly link your js file into your `blog.html`. You also did not include any of your individual blog page htmls. How are you getting your javascript to run? Reach out if you need help!

File: blog.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+              <li class="nav"><a href="resume.html">Resume</a></li>
+              <li class="nav"><a href="contact.html">Contact</a></li>
+            </ul>
+        </nav>
+        <main>
+            <h1 class="page-title">MY BLOG</h1>
+            Hello World! This is my blog.
+        </main>
+        <footer>© 2023 Dhanvi Ganti | All Rights Reserved</footer>
+    </body>
```
Comment (jlaksana) [inline blog.html:25]: There should be a `<script>` tag somewhere here that links to your js file.

Comment (drdonv) [reply blog.html:25]: Fixed! Please let me know if I have any other issues I need to fix, thank you so much for your help!

Comment (drdonv) [reply blog.html:25]: Added the biog list as well for the individual htmls

**Comment by jlaksana** [comment]
Looks good now. Please make a new PR for Milestone 2

**Pull request closed without merging**